doc/mk_sdcard_image.sh

```diff
@@ -579,10 +579,9 @@ if [ ! -z ${DOMF_START} ]; then
 fi
 if [ ! -z ${DOMA_START} ]; then
 	# simg2img is used only if we have android as guest
-	if [ $(dpkg-query -W -f='${Status}' android-tools-fsutils 2>/dev/null | grep -c "ok installed") -eq 0 ];
-	then
-		echo "Please install simg2img (in debian-based: apt-get install android-tools-fsutils). Exiting.";
-		exit;
+	if [ -z "`which simg2img`" ]; then
+		echo "Error: can't find simg2img. Please install it.";
+		exit 2
```

Comment:
why 2? is it used somewhere to control the return value?

Reply:
I see that script returns 1 if error occurred during operation, but 2 is returned if something was not found during initial checks.
Taking into account that surrounding blocks return 2 if can't find some domain, I decided to return 2 if required tool was not found. Just to be aligned with nearest lines.
At same time I assume it's ok to return 1 in case of any error.